openc3-cosmos-cmd-tlm-api/app/controllers/storage_controller.rb

```diff
@@ -99,11 +114,10 @@ def download_file
 
   def get_download_presigned_request
     return unless authorization('system')
-    bucket = OpenC3::Bucket.getClient()
     bucket_name = ENV[params[:bucket]] # Get the actual bucket name
     raise "Unknown bucket #{params[:bucket]}" unless bucket_name
     path = sanitize_path(params[:object_id])
-    bucket.check_object(bucket: bucket_name, key: path)
```

Comment:
I don't think we want the up to 3s wait for the object here. Just get a request and when you access it it will be 404 if not there.